crates/egui-winit/src/lib.rs

```diff
@@ -36,6 +36,9 @@ use winit::{
 };
 
 pub fn screen_size_in_pixels(window: &Window) -> egui::Vec2 {
+    #[cfg(target_os = "ios")]
+    let size = window.outer_size();
+    #[cfg(not(target_os = "ios"))]
     let size = window.inner_size();
```

Comment:
We should explain this better. Does this make sense?

```suggestion
    let size = if cfg!(target_os = "ios") {
        // `outer_size` Includes the area behind the "dynamic island".
        // It is up to the eframe user to make sure the dynamic island doesn't cover anything important.
        // That will be easier once https://github.com/rust-windowing/winit/pull/3890 lands 
        window.outer_size();
    } else {
		window.inner_size()
	};
```